ocs_ci/ocs/uninstall.py

```diff
@@ -0,0 +1,164 @@
+            sc_name_list.append(storage_class.get('metadata').get('name'))
+
+    # Query for PVCs and OBCs that are using the storage class provisioners listed in the previous step.
+    pvc_to_delete = []
+    pvc_name_list = []
+    for sc in sc_name_list:
+        pvc_to_delete.extend(get_all_pvcs_in_storageclass(sc))
+
+    # ignoring all noobaa pvcs & make name list
+    for pvc in pvc_to_delete:
+        if "noobaa" in pvc.name:
+            pvc_to_delete.remove(pvc)
+        else:
+            pvc_name_list.append(pvc.name)
+
+    pods_to_delete = []
+    all_pods = get_all_pods()  # default openshift-storage namespace
+    all_pods.extend(get_all_pods(namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE))
+    all_pods.extend(get_all_pods(namespace=constants.OPENSHIFT_MONITORING_NAMESPACE))
+
+    for pod_obj in all_pods:
+        try:
+            pvc_name = get_pvc_name(pod_obj)
+        except UnavailableResourceException:
+            continue
+        if pvc_name in pvc_name_list:
+            pods_to_delete.append(pod_obj)
+
+    log.info("Removing monitoring stack from OpenShift Container Storage")
+    remove_monitoring_stack_from_ocs()
+
+    log.info("Removing OpenShift Container Platform registry from OpenShift Container Storage")
+    remove_ocp_registry_from_ocs(config.ENV_DATA['platform'])
+
+    log.info("Removing the cluster logging operator from OpenShift Container Storage")
+    csv = ocp.OCP(
+        kind=constants.CLUSTER_SERVICE_VERSION,
+        namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
+    )
+    logging_csv = csv.get().get('items')
+    if logging_csv:
+        clusterlogging_obj = ocp.OCP(
+            kind=constants.CLUSTER_LOGGING, namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
+        )
+        clusterlogging_obj.delete(resource_name='instance')
+
+    log.info("deleting pvcs")
+    for pvc in pvc_to_delete:
+        log.info(f"deleting pvc: {pvc.name}")
+        pvc.delete()
+
+    log.info("deleting pods")
+    for pod in pods_to_delete:
+        log.info(f"deleting pod {pod.name}")
+        pod.delete()
+
+    log.info("removing rook from nodes")
+    nodes_list = get_labeled_nodes(constants.OPERATOR_NODE_LABEL)
+    for node in nodes_list:
+        log.info(f"removing rook from {node}")
+        ocp_obj.exec_oc_debug_cmd(node=node, cmd_list=["rm -rf /var/lib/rook"])
+
+    log.info("Delete the storage classes with an openshift-storage provisioner list")
+    for storage_class in sc_list:
+        log.info(f"deleting storage class {storage_class.get('metadata').get('name')}")
+        sc_obj = ocp.OCP(kind=constants.STORAGECLASS)
+        sc_obj.delete(resource_name=storage_class.get('metadata').get('name'))
+
+    log.info("unlabaling storage nodes")
+    nodes_list = get_all_nodes()
+    for node in nodes_list:
+        node_obj = ocp.OCP(kind=constants.NODE, resource_name=node)
+        node_obj.add_label(resource_name=node, label='cluster.ocs.openshift.io/openshift-storage-')
```

Comment:
replace `'cluster.ocs.openshift.io/openshift-storage-'` with `constants.OPERATOR_NODE_LABEL`

Reply:
will look ugly - `constants.OPERATOR_NODE_LABEL[:-3] + '-'`

Reply:
so please create a label for that